.github/workflows/push_attestation_image.yml

```diff
@@ -62,3 +62,15 @@ jobs:
           push: true
           tags: |
             ${{ secrets.REGISTRY_NAME }}.${{ secrets.REGISTRY_DOMAIN }}/attestation:${{ steps.get_image_tag.outputs.image_tag }}
+
+      - name: Run Attestation Tests
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          curl -L \
```

Comment:
should a `--fail` or `--fail-with-body` flag be added to the `curl` command? I think `curl` will treat an HTTP response with a 4xx or 5xx status code as successful. I don't think we'd want the workflow to proceed if the tests haven't been started.

Reply:
That works for me. I wasn't aware of that curl flag.